src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -154,7 +154,7 @@ namespace EvaporativeCoolers {
         // These are purposefully not in the header file as an extern variable. No one outside of this should
         // use these. They are cleared by clear_state() for use by unit tests, but normal simulations should be unaffected.
         // This is purposefully in an anonymous namespace so nothing outside this implementation file can use it.
-        bool InitEvapCoolerMyOneTimeFlag(true);
+        bool MySetPointCheckFlag(true);
```

Comment:
Just pulled a single variable up to the global space for convenience and to get rid of the static quickly.